speechbrain/processing/features.py

```diff
@@ -995,250 +996,240 @@ def forward(self, x):
 
 @register_checkpoint_hooks
 class InputNormalization(torch.nn.Module):
-    """Performs mean and variance normalization of the input tensor.
+    """Performs mean and variance normalization over the time dimension of the input.
+
+    Running mean and running variance calculation is done using Welford's Algorithm.
+
+    WARNING: at first, the running statistics do not represent the "true" mean
+    and variance, but are estimates based on the data seen so far. Once enough
+    data has been seen, the stats should closely approximate the "true" values.
+
+    WARNING: Using global normalization, the first call of `forward()` will
+    throw an error if no updates have been performed (including the current batch),
+    i.e. on first call the epoch >= update_until_epoch or in eval() mode.
 
     Arguments
     ---------
     mean_norm : True
-         If True, the mean will be normalized.
+        If True, the mean will be normalized.
     std_norm : True
-         If True, the standard deviation will be normalized.
-    norm_type : str
-         It defines how the statistics are computed ('sentence' computes them
-         at sentence level, 'batch' at batch level, 'speaker' at speaker
-         level, while global computes a single normalization vector for all
-         the sentences in the dataset). Speaker and global statistics are
-         computed with a moving average approach.
-    avg_factor : float
-         It can be used to manually set the weighting factor between
-         current statistics and accumulated ones.
-    requires_grad : bool
-        Whether this module should be updated using the gradient during training.
-    update_until_epoch : int
-        The epoch after which updates to the norm stats should stop.
+        If True, the variance will be normalized.
+    norm_type : str, default "global"
+        String parameter whose value defines how the statistics are computed:
+         * 'sentence' computes norms per utterance (no running stats)
+         * 'batch' computes norms per input tensor (no running stats)
+         * 'global' computes norms over all inputs (single mean, variance)
+         * 'speaker' - DEPRECATED
+    avg_factor : float, optional
+        Passing avg_factor is DEPRECATED as this exactly matches the
+        behavior of BatchNorm. To maintain this behavior, use
+        `speechbrain.nnet.normalization.BatchNorm1d(momentum=avg_factor)`.
+    length_dim : int, default 1
+        The dimension for which to mask out the padding positions.
+    update_until_epoch : int, default 2
```

Comment:
default is 2 not 1 as stated in the docstring.

Reply:
The code will stop when epoch <= update_until_epoch, so "2" means one epoch of updates.